src/redisai.c

```diff
@@ -578,9 +578,6 @@ int RedisAI_ScriptRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv
         return RedisAI_ScriptRun_IsKeysPositionRequest_ReportKeys(ctx, argv, argc);
     }
 
-    if (argc < 6)
```

Comment:
We still have a minimum arity, like key and fn_name

Reply:
this is checked further on the parsing function